client/pages/Assessment/SectionWrapper/SectionWrapper.tsx

```diff
@@ -22,6 +24,20 @@ const SectionWrapper: React.FC = (props) => {
     section: string
   }>()
 
+  const updateReviewSummaryEvent = Sockets.updateReviewSummaryEvent({ countryIso, assessmentName, cycleName })
+
+  useEffect(() => {
+    const updateReviewSummaryEventHandler = () => {
+      dispatch(ReviewActions.getReviewSummary({ countryIso, assessmentName, cycleName }))
+    }
+
+    SocketClient.on(updateReviewSummaryEvent, updateReviewSummaryEventHandler)
+
+    return () => {
+      SocketClient.off(updateReviewSummaryEvent, updateReviewSummaryEventHandler)
+    }
+  }, [])
```

Comment:
they go in the same useEffect with parameters countryIso, assessmentName, cycleName